README.md

```diff
@@ -1,2 +1,87 @@
-# Kanban-Board
-This a javaScript capstone project.. 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#GameLand
+
+## Description 
+>GameLand is an online game website that allows users to view different kind of games available 
+>and share their views on each game the like or dislike. Built with HTML/CSS , JAVASCRIPT,API.
+ 
+
+
+
+## Built With
+- Languages: HTML5, CSS5, JavaScript, API
+- VCS: GitBash
+- Package Manager: Node, NPM
+- Webpack
+-Following github flows standards.
+
+
+
+## Setup
+> to use the project To do list please follow :
+
+>Open GitHub and go to my GitHub repository name: GameLand.to clone. Click “Code” and copy the given URL.
+
+>click code and copy the URL.
+
+>Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+>Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+## Syntax:
+
+>$ git clone {repository URL}
+
+## Usage
+>open your directory GameLand/
+
+>use vs-code or any IDE to view line code .
+
+## use you preferd browser to run the file .html to get project website. to run in terminal / git :
+$ npm start
+
+## Screenshots
+
+![gameland](https://user-images.githubusercontent.com/104431739/188227473-56f89471-0681-4258-b6bc-eca9b6ea1f2f.PNG)
+
+![gameland 2](https://user-images.githubusercontent.com/104431739/188227570-35045011-ca58-49bc-9d23-ccad89ce3650.PNG)
+
+## Live demo 
+[Ckick here]( https://tarikbouari.github.io/GameLand/)
```

Comment:
[OPTIONAL]

- [ ] This link isn't working kindly add a working live demo link, because you'll use this project in the future to show your portfolio!